### PR TITLE
Support tiered

### PR DESCRIPTION
**Pull request by alphmth**

### CODE REVIEW

File: packages/test-suite/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	"main": "src/index.js",
 	"type": "commonjs",
 	"scripts": {
-		"test": "xo && nyc ava",
+		"test": "xo --fix && nyc ava",
```
Comment (jaredwray) [inline packages/test-suite/package.json:8]: @alphmth can you remove this?

File: packages/tiered/package.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@keyv/tiered",
+  "version": "3.0.1",
```
Comment (jaredwray) [inline packages/tiered/package.json]: Lets move the version to 1.0.0

Comment (jaredwray) [inline packages/tiered/package.json]: Lets move the version to 1.0.0

File: packages/tiered/src/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+export = KeyvTiered;
+declare class KeyvTiered extends EventEmitter {
+	constructor(options: KeyvTiered.Options);
+	set(key: string, value: string | Record<string, unknown> | undefined): Promise<any>;
+	get(key: string): Promise<string | undefined>;
+	getMany(keys: string[]): Promise<string[] | undefined>;
+	delete(key: string): boolean;
+	deleteMany(keys: string[]): boolean;
+	clear(): Promise<void>;
+	// Iterator(namespace: string | undefined): AsyncGenerator<any, void, any>;
```
Comment (jaredwray) [inline packages/tiered/src/index.d.ts]: Is Iterator not valid for this?

Comment (jaredwray) [inline packages/tiered/src/index.d.ts]: Is Iterator not valid for this?

**Review by jaredwray — CHANGES_REQUESTED** [review]
@alphmth - this is looking great and some really minor changes but this should be ready to go.

File: packages/tiered/src/index.d.ts (hunk 3 above)
Comment (stropicall) [reply packages/tiered/src/index.d.ts]: I haven't tried with an iterator. I will try and see

**Comment by mcfedr** [comment]
has this not been released yet?

**Comment by jaredwray** [comment]
The initial package is now live but the readme wont be live until end of week.

**Pull request merged**